editor/editor.tsx

```diff
@@ -947,6 +954,35 @@ function Editor(props: any) {
                                             }}
                                         />
                                     </div>
+                                    {/* {expertMode && false ? (
```

Comment:
Planning to open a separate PR to show hovering/clicking event data as a table in the Editor.